blog-collector/2025-12-15-otel.md

```diff
@@ -0,0 +1,10 @@
+---
+title: OpenTelemetry Collector
```

Comment:
I would format the title to match the other Collector release notes, like these:<br/>
<img width="500" alt="Screenshot 2025-12-11 at 3 26 00 PM" src="https://github.com/user-attachments/assets/ae209c21-0c9b-489e-8ddd-c1d786a3097f" />

So maybe something like...

```suggestion
title: New Log Severity Fields (OpenTelemetry Collector)
```